ui/app/peers/create/configuration/schema.ts

```diff
@@ -0,0 +1,20 @@
+import Joi from 'joi';
+import { PeerConfig } from './types';
+
+export const pgSchema = Joi.object({
+  host: Joi.string().required(),
+  port: Joi.number().integer().min(1).required(),
+  database: Joi.string().min(1).max(100).required(),
+  user: Joi.string().alphanum().min(1).max(64).required(),
+  password: Joi.string().min(1).max(100).required(),
+  transactionSnapshot: Joi.string().max(100).allow('').optional(),
+});
+
+export const checkFormFields = (peerType: string, config: PeerConfig) => {
+  switch (peerType) {
+    case 'POSTGRES':
+      return pgSchema.validate(config);
```

Comment:
nit: we are showing extra double quotes for error field
![image](https://github.com/PeerDB-io/peerdb/assets/39798643/c7cf741c-6129-43f6-94ee-64d7f262a451)

Reply:
I think this is fine as it quotes does make the error more clear in a way

Reply:
For the Name field double quotes were not there, which is why I pointed it out. Looks a bit inconsistent

Reply:
Hey Joban there are no quotes now. Thanks